tests/conftest.py

```diff
@@ -126,13 +131,14 @@ def mock_test_spotlight_opportunities() -> list[Opportunity]:
 
 
 @pytest.fixture
-def allowed_payloads() -> list[OpportunityRequest]:
+def allowed_payloads() -> list[OrderRequest]:
```

Comment:
I take it after the recent rework this is only used by order tests? Maybe we should give it a more explicit name if that's the case? Perhaps that's as simple as `allowed_order_requests`? Or maybe better would be moving it into the `order_test.py` module to show it is in fact only used there? (Perhaps it would be worthwhile to do an audit of `conftest.py` to find any other fixtures only used and expected to only be used by a single module and move them given the magnitude of recent changes, though that doesn't necessarily have to be part of this PR).

Reply:
renamed, but didn't move, as this will take a bit of disentangling the fixture chain